mathics/doc/common_doc.py

```diff
@@ -862,6 +862,27 @@ def get_tests(self) -> Iterator[Tests]:
 
 
 class Documentation:
+    """
+    `Documentation` describes an object containing the whole documentation system.
+
+    Documentation
+        |
+        +--------0> Parts
+                      |
+                      +-----0> Chapters
+                                 |
+                                 +-----0>Sections
```

Comment:
It is a little more complicated than this.  There can be a Guide Section in between Chapters an Section. When this happens, the "Chapter" name is the same as the "Guide Section name" so in a sense there is an elision here. But Chapter properties and attributes and Guide Sections attributes are not quite the same. 

I think that subsection is optional.

Reply:
> 
> OK, but a Guide Section is not just another kind of sections with some special features, that also belong to a Chapter? 

Something I think it useful to keep in mind with Guide Sections is that the level of nesting is increased by one.  From the bottom a section can have either a guide section parent or a chapter.